Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx

```diff
@@ -49,12 +49,10 @@ export const ObiEditor: FC<ObiEditorProps> = ({
   );
 
   const deleteLgTemplates = (lgTemplates: string[]) => {
-    const lgPattern = /\[(bfd\w+-\d+)\]/;
     const normalizedLgTemplates = lgTemplates
       .map(x => {
-        const matches = lgPattern.exec(x);
-        if (matches && matches.length === 2) return matches[1];
-        return '';
+        const lgTemplateRef = LgTemplateRef.parse(x);
+        return lgTemplateRef ? lgTemplateRef.name : '';
```

Comment:
We can leverage the 'Nullish Coalescing' feature in ts 3.7

Reply:
#1686